tests/unit/ui/components/TestLoadingText.js

```diff
@@ -0,0 +1,24 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import LoadingText from 'ui/components/LoadingText';
+
+
+describe('<LoadingText />', () => {
+  const render = (props = {}) => shallow(<LoadingText {...props} />);
+
+  it('renders LoadingText element with className', () => {
+    const root = render();
+    expect(root).toHaveClassName('LoadingText');
+  });
+
+  it('lets you set a fixed width', () => {
+    const root = render({ width: 55 });
+    expect(root.prop('style')).toMatchObject({ width: '55%' });
```

Comment:
I think it helps for maintenance to keep assertions isolated to exactly what they are testing. In this case, the test is *only* checking for the width style rule. It should be flexible enough to allow new style rules in the future without the test breaking.